lib/utils.js

```diff
@@ -54,9 +54,13 @@ var escaped_one_to_xml_special_map = {
 }
  
 function normalizeXmlIncludingCR(string) {
-    return string.replace(/([\&"<>\r])/g, function(str, item) {
-        return xml_special_to_escaped_one_map[item];
-    })
+  // replace \r\n lines with \n
+  // See https://www.w3.org/TR/2000/WD-xml-c14n-20000119.html#charescaping
+  // and https://www.w3.org/TR/2001/REC-xml-c14n-20010315#Examples
+  return string.replace(/(\r\n|\n)+/g, '\n')
```

Comment:
I want to convert multiple lines into a single line so to still do that this would be slightly simpler: `/(\r?\n)+/g`